src/components/NavigationPages/NavigationPages.jsx

```diff
@@ -17,32 +17,25 @@ const pages = [
 ];
 
 export const NavigationPages = () => {
-  const [page, setPage] = useState("");
   const dispatch = useDispatch();
   const handleCloseMainMenu = (event) => {
-    setPage(event.target.innerHTML);
-    dispatch(changePage(event.target.innerHTML));
-    // console.log(event.target.innerHTML);
+    dispatch(changePage(event.target.innerText));
```

Comment:
using this approach is not safe, better to do as we discussed and do following:
`<MenuItem key={page} onClick={() => handleCloseMainMenu(page)}>`
...
`const handleCloseMainMenu = (page) => {
    setPage(page);
    dispatch(changePage(page));
}`